packages/playground/src/utils/nodeSelector.ts

```diff
@@ -207,7 +209,9 @@ export function normalizeNodeFilters(
     country: options?.location.country,
     gateway: options?.gateway,
     healthy: true,
-    rentableOrRentedBy: filters.dedicated ? options?.twinId : undefined,
+    rentable: bothRentalFiltersActive ? undefined : filters.dedicated || undefined,
+    rentedBy: bothRentalFiltersActive ? undefined : filters.rentedBy ? options?.twinId : undefined,
```

Comment:
`filters.rentedBy` is containing the twin id already, right? i

Reply:
yes or undefined

Reply:
so i suggest 
```suggestion
    rentedBy: bothRentalFiltersActive ? undefined : filters.rentedBy || undefined,
```